### DISCUSSION TITLE
ImportError: cannot import name 'field_validator' from 'pydantic'

### DISCUSSION THREAD

**Discussion by Grjia** [pydantic/pydantic#9302] [Question]
pydantic2 on macos 
 ImportError: cannot import name 'field_validator' from 'pydantic' (/opt/miniconda3/envs/python3.12/lib/python3.12/site-packages/pydantic/__init__.cpython-312-darwin.so). Did you mean: 'root_validator'?

**Comment by cgiturtle** [comment]
It's been reported for 1.x versions:

https://stackoverflow.com/questions/77172420/importerror-cannot-import-name-field-validator-from-pydantic

https://stackoverflow.com/questions/78109048/error-cannot-import-name-field-validator-from-pydantic

**Comment by nikhilmanuWR** [comment]
@Grjia @cgiturtle 

Do we have any solution for this? Currently using pydantic 2.7.3

**Reply by SVyusti** [reply]
@dosu Do we have any solution for this? Currently using pydantic 2.7.3

**Reply by NikhilManu** [reply]
Hi @SVyusti ,
For me it was the problem with venv. 
Make sure the app is running in the venv. If the problem is still there.

Run the command "pip3 list" and check the version of pydantic from the list, most probably it should be a older version.

**Comment by Dev-iL** [comment]
Ran into this issue today. In my case it didn't have anything to do with incorrect versions. Instead, I defined my dataclass in a `dataclasses.py` file, which was shadowing the `dataclasses` import from the standard library. After renaming the file (module) the issue went away.
